packages/@aws-cdk/aws-ecr-assets/package.json

```diff
@@ -96,10 +97,13 @@
   "engines": {
     "node": ">= 10.3.0"
   },
+  "bundledDependencies": [
```

Comment:
Wondering if this is necessary. I do see we bundle `minimatch` in other place though i'm not sure why. Do you have some context of this?

Reply:
I tried without but it would not pass linter rules. The linter required that I add it to the module's package.json and then it also required that I add it as a hoisted package. I assume this is because someone might possible pull in the `aws-ecr-asset` module without the rest of the project?

Reply:
Hmm, I don't have knowledge/context on this. But if the linters require it, its probably for a good reason. All hail the linters 👍